repository-data/webfiles/src/main/resources/site/freemarker/common/scripts/init-table-sort.js.ftl

```diff
@@ -1,12 +1,29 @@
 <#ftl output_format="HTML">
 
 <script>
-    var tableHead = document.querySelector('thead');
+    // @TODO - rather than targeting *all* tables, we should
```

Comment:
looks good, just wondering if this "TODO" still needs to be there or whether it can be removed?

Reply:
The `TODO` has not been actioned, but in my opinion it's something that should be addressed at some point in the future.